ocf_datapipes/transform/numpy_batch/datetime_features.py

```diff
@@ -0,0 +1,70 @@
+"""Datapipes to add Sun position to NumpyBatch"""
+
+import numpy as np
+from torch.utils.data import IterDataPipe, functional_datapipe
+
+from ocf_datapipes.batch import BatchKey
+
+
+def _get_date_time_in_pi(dt):
```

Comment:
Could we perhaps add a small unit test for this?

Reply:
Yep good idea